PlanificatorCMD/Utils/IPresentationMapper.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace PlanificatorCMD.Utils
+{
+    interface IPresentationMapper
+    {
+    }
```

Comment:
1.Interface is not public.
2.Add public ICollection<PresentationTag> MapToPresentationTag (Add_PresentationVerb addPresentationVerb).